.travis.yml

```
@@ -2,14 +2,14 @@
 # * http://www.objc.io/issue-6/travis-ci.html
 # * https://github.com/supermarin/xcpretty#usage
 
-osx_image: xcode9.2
+osx_image: xcode11.4
```

It looks like Xcode 11.5 [showed up on Travis the other day](https://docs.travis-ci.com/user/reference/osx/#xcode-115); since we're changing Xcode versions here, should we move all the way to 11.5?

```suggestion
osx_image: xcode11.5
```

---

👍 Yea good call.